cmd/kubeadm/app/cmd/reset.go

```diff
@@ -66,13 +66,13 @@ type Reset struct {
 
 func NewReset(skipPreFlight bool, certsDir string) (*Reset, error) {
 	if !skipPreFlight {
-		fmt.Println("[preflight] Running pre-flight checks")
+		fmt.Println("[preflight] Running pre-flight checks.")
```

Comment:
If you look throughout kubeadm, it generally does not have a period at the end of log statements. I would argue for us to get rid of them instead of adding them.